cmd/termui/presenter/chainInfoGetters.go

```diff
@@ -187,20 +187,20 @@ func (psh *PresenterStatusHandler) GetTrieSyncNumProcessedNodes() uint64 {
 }
 
 // GetTrieSyncProcessedPercentage will return the number of processed nodes during trie sync
-func (psh *PresenterStatusHandler) GetTrieSyncProcessedPercentage() uint64 {
+func (psh *PresenterStatusHandler) GetTrieSyncProcessedPercentage() (uint64, bool) {
```

Comment:
... or could have used an optional uint64 struct that already has the IsSetValue field

Reply:
changed to use OptionalUint64